recipes/valkey-py/recipe.yaml

```diff
@@ -0,0 +1,43 @@
+context:
+  name: "valkey"
+  version: "6.1.0"
+
+package:
+  name: "valkey-py"
+  version: "${{ version }}"
+
+source:
+  url: https://github.com/valkey-io/valkey-py/archive/refs/tags/v${{ version }}.zip
```

Comment:
Any reason for pulling it from Github vs the release on pypi?
https://pypi.org/project/valkey/